CONTRIBUTING.md

```diff
@@ -108,6 +108,26 @@ $ cargo version
 cargo 1.69.0 (6e9a83356 2023-04-12)
 ```
 
+## Build
+
+### Compile
+
+```shell
+make build
+```
+
+### Lint
+
+```shell
+make check
+```
+
+### Test
+
+```shell
+make test
+```
```

Comment:
Instead of using headers, how about using item lists:
```
* To invoke a build: `make build`
* To check code styles: `make check`
* To run tests: `make test`
```

Reply:
I'm trying to follow https://py.iceberg.apache.org/contributing/ but not sure yet what to add else for each command.

Reply:
Currently we don't have so much commands, so I think the list format would be more concise? We can always enrich these when necessary.